gh-docs/developer-guide/integration-studio-catalogue.md

```diff
@@ -150,6 +150,33 @@ Integration Studio has been released with the following package components.
 
 3. Clone Integration Studio GitHub repository https://github.com/wso2/integration-studio.git. and proceed with **main** branch.
 
+---
+**If you are using maven version 3.8.1 or higher**
+
+From maven version 3.8.1, maven default settings file updated with http blocker mirror. To prevent maven from blocking the http transfer we can 
```

Comment:
file **is** updated with http blocker mirror.

Reply:
Fixing in : [1128](https://github.com/wso2/integration-studio/pull/1128)